cvmfs/mountpoint.cc

```diff
@@ -682,17 +686,26 @@ CacheManager *FileSystem::SetupExternalCacheMgr(const string &instance) {
 }
 
 
-CacheManager *FileSystem::SetupPosixCacheMgr(const string &instance) {
+CacheManager *FileSystem::SetupPosixCacheMgr(const string &instance,
+                                             bool use_refcount) {
   PosixCacheSettings settings = DeterminePosixCacheSettings(instance);
+  CacheManager* cache_mgr;
```

Comment:
I think this needs to be a `UniquePtr<CacheManager>`. Or we need to call `Release()` on the object returned from `[PosixCacheManager|PosixRefcountCacheManager]::Create()`. Otherwise the cache manager will be immediately deconstructed.